app/views/govuk_publishing_components/components/_share_links.html.erb

```diff
@@ -42,6 +43,26 @@
                 'socialNetwork': link[:icon],
                 'socialTarget': link[:href]
               }
+              ga4_link_data = {
```

Comment:
Non-blocking comment - it might be good to abstract this ruby code out into a helper method to make the template a bit cleaner, as we do with some other components.